app/templates/browser-support.hbs

```diff
@@ -3,6 +3,40 @@
   <section aria-labelledby="section-browser-support-policy">
     <h1 id="section-browser-support-policy">Ember.js Browser Support Policy</h1>
 
+    <h2>Ember 6.0.0</h2>
+
+    <p>
+      In Ember 6.0.0, the framework supports the following major browsers:
```

Comment:
ember 6 isn't out yet, so I'm not sure what we want to do here -- or maybe this whole section could be omitted until ember 6?

especially since the versions below on lines 16-19 are likely to change

Reply:
I think maybe omit this section until 6.0 exists -- the policy below is enough